components/scream/cime_config/config_compsets.xml

```diff
@@ -15,12 +15,24 @@
     <support_level>Experimental, under development</support_level>
   </compset>
 
+  <compset>
+    <alias>F2010-SCREAM-SA</alias> <!-- SCREAM standalone -->
+    <lname>2010_SCREAM_SLND_SICE_SOCN_SROF_SGLC_SWAV</lname>
+    <support_level>Experimental, under development</support_level>
+  </compset>
+
   <compset>
       <alias>F2000SCREAMv1</alias> <!-- Coupled land-atm compset (using SCREAMv1), 2000 initial conditions -->
       <lname>2000_SCREAM_ELM%SPBC_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV_SIAC_SESP</lname>
       <support_level>Experimental, under development</support_level>
   </compset>
 
+  <compset>
```

Comment:
We should also add a test for this new compset in this PR. See `cime_config/tests.py` at the top-level of the e3sm repo for where to add this. I might actually suggest we replace the F2000 tests with these F2010 compsets, and abandon the F2000 altogether.

Reply:
I think the test should go in this PR that adds the compset, we don't want to add things that are not tested. If you want to keep the PR BFB, we could add the tests alongside the others, and then in a subsequent PR remove testing of the F2000 tests. Replacing F2000 with F2010 will have the side effect of making this PR look like it's not BFB I think, since there will be missing baselines and missing tests.

Reply:
Okay, kept the F2000 test and added the F2010 test.